sandbox.go

```diff
@@ -691,6 +691,9 @@ func (sb *sandbox) EnableService() error {
 func (sb *sandbox) DisableService() error {
 	logrus.Debugf("DisableService %s START", sb.containerID)
 	for _, ep := range sb.getConnectedEndpoints() {
+		if err := ep.deleteServiceInfoFromCluster(sb, "DisableService"); err != nil {
```

Comment:
agreed on that. From what I see in here - I see that service is disabled first before task shutdown. The race condition was seen on the remote nodes when this happens ? Just making sure I incorporate the scenario in my testing.   
With the current code we are basically adding 2 second delay to do nothing so we have increased our convergence time.